### DISCUSSION TITLE
MONSA Online

### DISCUSSION THREAD

**Discussion by fernanto22** [biosaxs-com/atsas-community#1968] [Ab initio shape determination] [answered]
Hi, I'm new to SAXS data analysis and I'm trying to analyze a protein complex. 
I generated .dat files via primus and obtained reasonable looking envelops via DAMMIF and GASBOR. When I input these as data files into the MONSA online, I am getting model that look like spheres of balls. 
Would anyone know to fix this problem please.
Thanks for your help in advance.

Best, Nanto

Attachments:

![Monsa Online](https://github.com/user-attachments/assets/bd7e43e8-00f3-492e-95b2-7b8eafa16524)
![sample2 model](https://github.com/user-attachments/assets/06a28082-a75f-45e3-915d-fee6bcdfedbe)
[Sample2.log](https://github.com/user-attachments/files/17657597/Sample2.log)

**Comment by maxim-biosaxs** [comment, marked as answer]
Judging from the log file:
...
Angular units 1/nm transformed to 1/Angstrom
...
S-limits for the master curve:   0.000000  0.028708
...

It seems that your data is in fact in A^-1 but it is additionally converted as if it was in nm^-1.
Please try to click on "show advanced settings" and specify angular units as 1/A

**Reply by fernanto22** [reply]
Thank you! it works fine now.
